src/distributed_join.cuh

```diff
@@ -203,16 +252,16 @@ all_to_all_comm_single_batch(
 void
 inner_join_func(
```

Comment:
Sounds like we could use some doxygen comments here - lots of arguments, and it's worth clarifying that this function may create threads internally.

Reply:
Additional documentation is added in https://github.com/rapidsai/distributed-join/pull/27/commits/6fbc2c5c2fd36d7e0432360ebb3eb9982f65f66a